modules/checking-mco-node-status.adoc

```diff
@@ -2,83 +2,151 @@
 //
 // * post_installation_configuration/machine-configuration-tasks.adoc
 
-:_mod-docs-content-type: PROCEDURE
+:_mod-docs-content-type: CONCEPT
 [id="checking-mco-node-status_{context}"]
-= Checking machine config node status
+= About checking machine config node status
 
-During updates you might want to monitor the progress of individual nodes in case issues arise and you need to troubleshoot a node.
+If you make changes to the control plane or worker nodes in your cluster, for example by using a machine config or kubelet config, you can get detailed information about the progress of the node updates by using the machine config nodes custom resource. Any change that results in a new machine config triggers the node update process.
 
-To see the status of the Machine Config Operator (MCO) updates to your cluster, use the following `oc` commands:
+The `MachineConfigNode` custom resource allows you to to monitor the progress of individual node updates, which can be helpful with troubleshooting if one of the nodes has an issue during the update. The custom resource reports where in the update process the node is at the moment, the phases that have completed, and the phases that are remaining.
 
-:FeatureName: Improved MCO state reporting
+:FeatureName: The machine config node custom resource
 include::snippets/technology-preview.adoc[]
 
-.Prerequisites
-* You set `featureSet: TechPreviewNoUpgrade` in your `FeatureGate` custom resource (CR) to enable the feature set capability for your cluster.
+The node update process consists of the following phases and subphases that are tracked by the machine config node custom resource:
+
+* *Update Prepared*. The MCO stops the configuration drift monitoring and verifies that the newly-created machine config can be applied to a node.
+** *UpdateCompatible*
+* *Update Executed*. The MCO cordons and drains the node, and applies the new machine config to the node files and operating system, as needed. It contains the following sub-phases:
+** *Cordoned*
+** *Drained*
+** *AppliedFilesAndOS*
+* *PinnedImageSetsProgressing*
+* *PinnedImageSetsDegraded*
+* *Update Post update action*. The MCO reboots the node or reloads CRI-O, as needed.
+** *RebootedNode*
+** *ReloadedCRIO*
+* *Update Complete*. The MCO uncordons the nodes, updates the node state to the cluster, and resumes producing node metrics.
+** *Updated*
+** *Uncordoned*
+* *Resumed*. The MCO restarts config drift monitor. The node returns to operational state.
+
+As the update moves through these phases, you can query the `MachineConfigNode` custom resource, which reports one of the following conditions for each phase:
+
+* `True`. The phase is complete on that node or the MCO has started that phase on that node.
```

Comment:
We could include a note explaining that once the nodes is fully updated  all values are reset.